content/guides/dashboard/github-integration.md

```diff
@@ -88,6 +107,78 @@ you choose **All repositories**.
 
 3. Click the **Install** button to complete the installation.
 
+### Cypress GitHub Enterprise app installation process
+
+For GitHub Enterprise the installation is a little more involved, you need to
+create a new GitHub App, copy various settings and credentials from the new app,
+paste them into the Dashboard and go through the GitHub app activation process.
+
+1. With your Dashboard organization's GitHub Enterprise page open, open a new
+   tab or browser window and browse to your GitHub Enterprise site. Navigate to
+   your GitHub organization -> **Settings** -> **Developer settings** ->
+   **GitHub Apps**. Do **NOT** go to **OAuth Apps**. Click the **New GitHub
+   App** button.
+
+2. Fill in the **Register new GitHub app** form.
+
+   - Enter a **GitHub App name**. Name may contain only dashes, letters and
+     numbers, **no spaces**.
+   - Enter the Cypress Dashboard **Homepage URL**, https://dashboard.cypress.io
+   - Enter the **Callback URL** and **Setup URL**,
+     https://dashboard.cypress.io/apps/github/callback
+
+3. Fill in the **Webhook** form.
+
+   - Enter the **Webhook URL**, https://dashboard.cypress.io/webhooks/github-app
+   - Copy and paste your Cypress organization's **Webhook secret** from the
+     Dashboard into the GitHub Webhook form.
```

Comment:
You can copy this directly from the Dashboard GitHub Enterprise configuration page.